qiskit/_compiler.py

```diff
@@ -279,6 +294,54 @@ def load_unroll_qasm_file(filename, basis_gates='u1,u2,u3,cx,id'):
     return circuit_unrolled
 
 
+def best_subset(backend, n_qubits):
```

Comment:
Would it be possible to have a couple of tests that exercise this function (or that invoke it trough `compile()`)? Probably using the values that were reported in the initial issue or some edge cases would be a good idea.

Reply:
I am trying to find a short test that raises an error.

Reply:
Done.